portals/new_admin/portals/client/public/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <title>WSO2 API Manager</title>
+    <script type="text/javascript" src="/Settings.js"></script>
+  <script defer src="/bundle.js"></script></head>
+  <body>
+    <div id="root"></div>
+  </body>
+</html>
```

Comment:
Shall we add an EOF new line here as well?

Reply:
Currently, there is no such file.